packages/config/src/assets.ts

```diff
@@ -220,7 +220,7 @@ export const unit = new Asset({
   originSymbol: 'Unit',
 });
 
-export const usdc = new Asset({
+export const usdcwh = new Asset({
```

Comment:
@ekenigs Do you think we should change this? On one side it would make sense to change it because it should be the correct name.

On the other hand I guess it would be a breaking change, though small.

Reply:
I think we should do it if it makes sense.